spec/models/person_spec.rb

```diff
@@ -194,7 +194,7 @@
       it "sends call to update NationBuilder" do
         expect_any_instance_of(Person).to receive(:update_nation_builder).and_call_original
         expect(NbPersonPushJob).to receive(:perform_later)
-          .with("email" => "user@example.com", "phone"=>"510-555-1234")
+          .with(email: 'user@example.com', phone: '510-555-1234')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.